db/connection.js

```diff
@@ -0,0 +1,4 @@
+var seedCompliments = require("./compliments.json");
```

Comment:
Since you are not actually using a database, I might call this file something else or just export the compliments.json directly